src/main/java/com/legendary/coffeeShop/controller/form/ComponentForm.java

```diff
@@ -0,0 +1,22 @@
+package com.legendary.coffeeShop.controller.form;
+
+import com.legendary.coffeeShop.dao.entities.ComponentStatus;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+public class ComponentForm {
+
+    private int id;
+
+    private int amount;
+
+    private String name;
+
+    private double price;
+
+    private ComponentStatus status;
+
+    private int productTypeId;
```

Comment:
Please change the type to string and ass "type" parameter (or any different name).
The "type" will us as identify of the component.

Reply:
whats the meaning of type?

Reply:
As I said we need identify for the component e.g. name can be like "Cherry Tomatoes" and this name will display on the client side but the type will be something like "cherry_tomato" that can use for identity (this column don't have to be called type if you have a better name....)

Reply:
why do we need the identity? why id is not enough?